frontend/packages/vsphere-plugin/src/components/VSphereConnectionForm.tsx

```diff
@@ -0,0 +1,238 @@
+import * as React from 'react';
+import { Form, FormGroup, TextInput } from '@patternfly/react-core';
+import { Trans, useTranslation } from 'react-i18next';
+import { useK8sModel } from '@console/dynamic-plugin-sdk/src/api/core-api';
+import { PopoverHelpButton } from '../PopoverHelpButton';
+import { useConnectionFormContext } from './ConnectionFormContext';
+import { initialLoad } from './initialLoad';
+import { VSphereConnectionProps } from './types';
+
+import './VSphereConnectionForm.css';
+
+export const VSphereConnectionForm: React.FC<VSphereConnectionProps & { formId?: string }> = ({
+  cloudProviderConfig,
+  formId,
```

Comment:
why is `formId` passed as prop ? It does not look like something which would change